ring-core/src/ring/middleware/resource.clj

```diff
@@ -58,6 +58,7 @@
   ([handler root-path]
    (wrap-resource handler root-path {}))
   ([handler root-path options]
-   (if (:prefer-handler? options)
-     (wrap-resource-prefer-handler   handler root-path options)
-     (wrap-resource-prefer-resources handler root-path options))))
+   (let [root-path (clojure.string/replace root-path #"/$" "")]
```

Comment:
Can you `require` this instead of using it bare? I don't know if we can rely on `clojure.string` being loaded by default.